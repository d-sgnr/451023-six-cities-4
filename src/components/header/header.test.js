import React from "react";
import renderer from "react-test-renderer";
import Header from "./header.jsx";

const children = <div className="children-component" />;

it(`Header should be rendered correctly`, () => {
  const tree = renderer.create(
      <Header>
        {children}
      </Header>).toJSON();

  expect(tree).toMatchSnapshot();
});
