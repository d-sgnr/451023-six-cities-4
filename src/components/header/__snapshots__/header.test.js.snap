// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should be rendered correctly 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="children-component"
      />
    </div>
  </div>
</header>
`;
